pydantic/fields.py

```diff
@@ -359,6 +369,8 @@ def Field(
     :param default_factory: callable that will be called when a default value is needed for this field
       If both `default` and `default_factory` are set, an error is raised.
     :param alias: the public name of the field
+    :param validation_alias: the alias(es) to use to find the field in the validation data
```

Comment:
```suggestion
    # TODO: Add documentation reference for non-str alias variants
    :param validation_alias: the alias(es) to use to find the field value during validation
```
We will need to document this better at some point but